nservicebus/upgrades/7to8/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Upgrade Version 7 to 8
+summary: Instructions on how to upgrade NServiceBus from version 7 to version 8.
+component: Core
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 7
+ - 8
+---
+
+## Obsoleted APIs
```

Comment:
Do we need this section? perhaps just name the one below "New gateway persistence API" or something?

Reply:
I have no idea how upgrade guides are structured, they seem to be very inconsistent. I'm fine with dropping the section though.